src/Symfony/Component/Cache/Adapter/TagAwareAdapter.php

```diff
@@ -177,9 +177,10 @@ public function hasItem($key)
         }
 
         foreach ($this->getTagVersions([$itemTags]) as $tag => $version) {
-            if ($itemTags[$tag] !== $version && 1 !== $itemTags[$tag] - $version) {
-                return false;
+            if ($itemTags[$tag] === $version || \is_int($itemTags[$tag] && \is_int($version) && 1 === $itemTags[$tag] - $version)) {
```

Comment:
`is_int($itemTags[$tag]` is missing its ending bracket

Reply:
Thank you @nicolas-grekas!